src/__tests__/requests.test.ts

```diff
@@ -39,6 +39,18 @@ const getSimpleT: GetSimpleT = {
   url: params => `/api/v1/simples/${params.id}`
 };
 
+const getSimpleTP: GetSimpleT = {
+  headers: () => ({
+    Authorization: "Bearer: 123"
+  }),
+  method: "get",
+  query: params => ({ param1: `${params.p1}`, param2: params.p2 }),
+  response_decoder: basicResponseDecoder(SimpleModel, _ => {
```

Comment:
no need for curly braces

```suggestion
  response_decoder: basicResponseDecoder(SimpleModel, _ => processedValue)
```